csharp/src/Google.Protobuf/MessageExtensions.cs

```diff
@@ -109,9 +109,7 @@ public static byte[] ToByteArray(this IMessage message)
         {
             ProtoPreconditions.CheckNotNull(message, "message");
             byte[] result = new byte[message.CalculateSize()];
-            CodedOutputStream output = new CodedOutputStream(result);
-            message.WriteTo(output);
-            output.CheckNoSpaceLeft();
```

Comment:
what happens with the CheckNoSpaceLeft() check here?

Reply:
The span based WriteTo() extension also performs that check,
https://github.com/protocolbuffers/protobuf/blob/949caa7d66e4c1f0f4afe7bddabe5416eaaf4285/csharp/src/Google.Protobuf/MessageExtensions.cs#L180-L187